src/formatDate.js

```diff
@@ -8,7 +8,34 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const oldSeparator = fromFormat.slice(3);
```

Comment:
The `slice` method is being used incorrectly to get the separator. Since `fromFormat` and `toFormat` are arrays, you should be using the index to get the separator, not `slice`. Use `fromFormat[3]` to get the old separator.